state-chain/node/src/chain_spec.rs

```diff
@@ -97,6 +97,8 @@ pub struct StateChainEnvironment {
 	sol_vault_address: SolAddress,
 	sol_genesis_hash: Option<SolHash>,
 	sol_usdc_address: SolAddress,
+	sol_durable_nonces_and_accounts: [DurableNonceAndAccount; 7], /* we inject 7 nonce accounts
```

Comment:
Do we always need 7 accounts? Event on a localnet?

Reply:
Yes, this is the fixed number of nonce accounts we want to use. We'll use the same number of accounts on all networks to make testing similar to real world scenario.